runtime.txt

```diff
@@ -0,0 +1 @@
+python-3.7
```

Comment:
This should just be `x.y` as follows:
```suggestion
3.7
```

Also [remove the blank line at the end of the file](https://docs.netlify.com/configure-builds/manage-dependencies/#python).